packages/request-manager/src/interceptor.ts

```diff
@@ -108,22 +96,25 @@ const overloadHttpRequest = (
         (!options || typeof options === 'function')
     ) {
         const isTorEnabled = interceptorOptions.getIsTorEnabled();
-        // get authorization data from request headers
-        const overloadedOptions = getAuthorizationOptions(url);
+        const isTorRequired = getIsTorRequired(url);
+        const overloadedOptions = url;
```

Comment:
Here the reason to do the assignment `const overloadedOptions = url;` is just for readability right?
Since after the `if` condition we know this is an object and therefore they are options and not `url`.

Reply:
Exactly, it's more like renaming in order not to confuse anyone.